pcsx2-qt/Debugger/DebuggerSettingsManager.cpp

```diff
@@ -0,0 +1,163 @@
+// SPDX-FileCopyrightText: 2002-2024 PCSX2 Dev Team
+// SPDX-License-Identifier: LGPL-3.0+
+
+#include "DebuggerSettingsManager.h"
+
+#include <QtCore/QJsonDocument>
+#include <QtCore/QJsonObject>
+#include <QtCore/QJsonArray>
+#include <QtCore/QFile>
+
+#include "common/Console.h"
+#include "fmt/core.h"
+#include "VMManager.h"
+#include "Models/BreakpointModel.h"
+
+std::mutex DebuggerSettingsManager::writeLock;
+
+QJsonObject DebuggerSettingsManager::loadGameSettingsJSON() {
+	std::string path = VMManager::GetDebuggerSettingsFilePathForCurrentGame();
+	QFile file(QString::fromStdString(path));
+	if (!file.open(QIODevice::ReadOnly))
+	{
+		Console.WriteLn(fmt::format( "Debugger Settings Manager: No Debugger Settings file found for game at: '{}'", path));
```

Comment:
```suggestion
		Console.WriteLnFmt("Debugger Settings Manager: No Debugger Settings file found for game at: '{}'", path);
```

Added this recently. Can be done for the others in this file too.